onnx/mapping.py

```diff
@@ -21,7 +21,7 @@
     int(TensorProto.COMPLEX64): np.dtype('complex64'),
     int(TensorProto.COMPLEX128): np.dtype('complex128'),
     int(TensorProto.UINT32): np.dtype('uint32'),
-    int(TensorProto.UINT64): np.dtype('uint64'),
+    int(TensorProto.POSIX_DATETIME): np.dtype('uint64'),
```

Comment:
why remove UNIT64 case?

Reply:
Oversight

---
In reply to: [344909641](https://github.com/onnx/onnx/pull/2437#discussion_r344909641) [](ancestors = 344909641)